core/evancore.vue.libs/src/components/forms/textarea/textarea.vue

```diff
@@ -37,3 +45,7 @@ import Component from './textarea';
 
 export default Component;
 </script>
+
+<style lang="scss" scoped>
+  @import './textarea.scss'
```

Comment:
```suggestion
  @import './textarea.scss';
```
This should be marked in VSCode?